### ISSUE TITLE
Add Libpanel to Packages

### ISSUE THREAD

**Issue by Zacharymk1213** [msys2/MSYS2-packages#3134]
Libpanel is required for certain packages such as Gnome Builder which is in the process of being ported. I was able to compile it with a subproject. To make it'd easier it'd be better to package it officially.

**Comment by Biswa96** [comment]
It can not be built right now. The project requires gtk4 version 4.7 minimum. The mingw gtk4 package is 4.6.6. I have to do this workaround to build it.

```diff
--- a/meson.build
+++ b/meson.build
@@ -27,7 +27,7 @@ vapidir = join_paths(datadir, 'vala', 'vapi')
 
 # Check dependencies
 glib_req_version = '2.72'
-gtk_req_version = '4.7'
+gtk_req_version = '4.6'
 adwaita_req_version = '1.0'
 
 glib_req = '>= @0@'.format(glib_req_version)
```

Would this workaround be correct?

**Comment by Zacharymk1213** [comment]
The subproject was executed after using another subproject to update GTK4
for me when I compiled it.

**Comment by Biswa96** [comment]
Yes. But gtk4 version 4.7.1 is marked as unstable in https://www.gtk.org/.

**Comment by Zacharymk1213** [comment]
Msys2's policy is to only add stable packages?

**Comment by Biswa96** [comment]
I am not sure how much "unstable" that 4.7.1 version of gtk4 is. The ArchLinux distribution even has [gtk4](https://archlinux.org/packages/extra/x86_64/gtk4/) version 4.6.6.

**Comment by Biswa96** [comment]
BTW, the gtk version was bumped in this commit https://gitlab.gnome.org/GNOME/libpanel/-/commit/b2daca3149c44e2bbaacad6bdb4d5f2609c43c83. It may be possible to build the package before that commit to be compatible with current mingw gtk4 4.6.6 package. What do you think about this idea?

**Comment by Zacharymk1213** [comment]
Sounds good.
